### DISCUSSION TITLE
WARNING: Circular directory structure

### DISCUSSION THREAD

**Discussion by ASF1967** [trapexit/mergerfs#928] [Q&A]
Hello, I don't know if this is an issue or not., but I get the following message when I run a du command against a directory in mergerfs:

root@smicro:/storage/ml36# du -sh Seagate4TB/
du: WARNING: Circular directory structure.
This almost certainly means that you have a corrupted file system.
NOTIFY YOUR SYSTEM MANAGER.
The following directory is part of the cycle:
  'Seagate4TB/8710p/disk_c/Documents and Settings/Albert/Dati applicazioni/Macromedia/Flash Player/#SharedObjects/U9JLN8WY/skype.com/#ui'

du: WARNING: Circular directory structure.
This almost certainly means that you have a corrupted file system.
NOTIFY YOUR SYSTEM MANAGER.
The following directory is part of the cycle:
  'Seagate4TB/8710p/disk_c/Documents and Settings/Albert/Dati applicazioni/Macromedia/Flash Player/#SharedObjects/U9JLN8WY/www.hallmark.com'

du: WARNING: Circular directory structure.
This almost certainly means that you have a corrupted file system.
NOTIFY YOUR SYSTEM MANAGER.
The following directory is part of the cycle:
  Seagate4TB/data

2.6T    Seagate4TB/

If I run the du command directly on the physical disk I have no issues:

root@smicro:/mnt/DiskPools/NetApp01_07/ml36# du -sh Seagate4TB/
2.8T    Seagate4TB/

But if I run it agaist the virtual disk created I got the above messages, and it seems it does not see some of the data , reporting a total of  2.6Tb instead of 2.8Tb.

Is that normal?
Reagards
Albert

**Comment by trapexit** [comment]
Please provide the details requested in the issue template or support docs otherwise I really can't help.

**Reply by ASF1967** [reply]
Sorry I miss the "issue" session, but I don't find the template.
Sorry I am new to github.
Regards
Albert

**Reply by ASF1967** [reply]
Find it! I am in the process of migration the server to a new version, the data disk are in a external sas enclosure.
If after the update and reconnection of the disks  the bug is still there I will open an issue report.
Sorry again for the misunderstanding.
Regards
Albert
